### PR TITLE
Complete all Tests for Genomix Node, Now Genomix node is safe temparally

### PR DESCRIPTION
**Pull request by Nan-Zhang**
@JavierJia @jakebiesinger @anbangx Please help me to check these 36 test cases for covering all our Genomix Node. They all can be passed now based on our correcting offset.

I think our Genomix Node is temporarily safe now. If you guys are doubt to some function's correctness, please let me know, I will add more test cases for it.

### CODE REVIEW

File: genomix/genomix-data/src/main/java/edu/uci/ics/genomix/type/Node.java [hunk 1]
```diff
@@ -109,6 +109,11 @@ public boolean hasNext() {
 
                 @Override
                 public NeighborInfo next() {
+                    if(info == null){
+                        info = new NeighborInfo(et, it.next());
+                    }else{
```
Comment (jakebiesinger) [inline genomix/genomix-data/src/main/java/edu/uci/ics/genomix/type/Node.java]: what is this `else` block for?

Comment (jakebiesinger) [inline genomix/genomix-data/src/main/java/edu/uci/ics/genomix/type/Node.java]: Also don't forget to reformat your code!

**Comment by jakebiesinger** [comment]
As a general comment, you shouldn't move a file and change it in the same commit, otherwise, it's very hard to track any changes you've made to the file!

Also, it looks like you need to reformat the NodeTest.java file.  Hit Ctrl+Shift+F or whatever the mac equivalent is.

File: genomix/genomix-data/src/main/java/edu/uci/ics/genomix/type/Node.java (hunk 1 above)
Comment (Nan-Zhang) [inline genomix/genomix-data/src/main/java/edu/uci/ics/genomix/type/Node.java]: (Done) already delete the un-use "else" block which jake pointed out

Comment (Nan-Zhang) [inline genomix/genomix-data/src/main/java/edu/uci/ics/genomix/type/Node.java]: (Done) already format the piece code

**Comment by JavierJia** [comment]
I think this pull request is ready to merge :+1: 
@jakebiesinger What's your opinion ?

**Comment by Nan-Zhang** [comment]
Can this branch be allowed to be merged into our main branch? @jakebiesinger @JavierJia @anbangx

**Comment by jakebiesinger** [comment]
LGTM.

**Pull request merged**